okio/src/jvmTest/kotlin/okio/FileHandleFileSystemTest.kt

```diff
@@ -53,3 +58,12 @@ class FileHandleFileSystemTest : AbstractFileSystemTest(
     }
   }
 }
+
+class FileHandleNioFileSystemWrapperFileSystemTest : AbstractFileSystemTest(
+  clock = Clock.System,
+  fileSystem = FileHandleTestingFileSystem(Jimfs.newFileSystem(Configuration.unix()).asOkioFileSystem()),
+  windowsLimitations = Path.DIRECTORY_SEPARATOR == "\\",
```

Comment:
I think you want this to always be `false` since you're creating a Unix-like FS with JimFS. Currently this value presumably reflects the default file system value.

Reply:
yes, fixed!
Maybe I could setup a windows or osx configuration depending on the platform we're running on..... but not now